modules/cnf-about-topology-aware-lifecycle-manager-config.adoc

```diff
@@ -16,6 +16,9 @@ The {cgu-operator-first} manages the deployment of {rh-rhacm-first} policies for
 * The order of policies remediated to the cluster
 * The assignment of a canary cluster
 
-For {sno}, the {cgu-operator-first} can create a backup of a deployment before an upgrade. If the upgrade fails, you can recover the previous version and restore a cluster to a working state without requiring a reprovision of applications.
+For {sno}, the {cgu-operator-first} offers the following features:
+
+* create a backup of a deployment before an upgrade
+* pre-caching images for clusters with limited bandwidth
```

Comment:
Capitalize first letter in each list entry ([ISG](https://www.ibm.com/docs/en/ibm-style?topic=format-lists#capitalization-in-lists))